Loop/Views/BolusEntryView.swift

```diff
@@ -202,8 +202,9 @@ struct BolusEntryView: View, HorizontalSizeClassOverride {
                 }
             }
             .onAppear {
-                // After the row is first made visible, make the text field the first responder
-                DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(10)) {
+                // After the row is first made visible, make the text field the first responder.
+                // Long delay is needed here to allow the view to scroll after the keyboard is presented
+                DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(200)) {
```

Comment:
@nhamming Hm.  I’m not sure I understand how this fixes a crash.  The comment explains what it is doing but not why _not_ delaying causes a crash. I’d have to see the stack trace to understand this better.

Reply:
@rickpasetto sure take a look.

Reply:
I don't. have an iPod touch... is that required to reproduce, @nhamming  ?
If so, can you put the stack trace into the bug, please?